### ISSUE TITLE
Installation and setup issue

### ISSUE THREAD

**Issue by edveal** [hgboldt/Tangled-Web#7]
I am having some difficulty installing and getting this up and running. I think I have followed your directions, but I find them lacking in specifics. Whenever I try to enter the "Instance Id" I am told it can't find the instance. I suspect I am either using the wrong "Instance Id" or I have placed something in the wrong directory. Might you provide some assistance?

Ed

**Comment by hgboldt** [comment]
I need more detail. What specific component are you using, the Gramps plugin or the WordPress plugin? Are you using the main branch or the dev branch? What specific message are you getting? If you're in WordPress, is your WP installed in the server root directory? A screenshot may be useful in understanding what's happening.

**Comment by edveal** [comment]
I had no problem installing and running the Gramps Plugin. The issue is in my WordPress setup. I am using a standard LAMP stack install with the WP site and plugin installed here: /var/www/html/wp-content/plugins. I have zipped and copied the export from Gramps to /var/www/html/wp-content/plugins as well as my user/home/. 

When I attempt to create the Tangled Web instance from the WP dashboard I can't seem to identify the proper "Instance Id" and or "Directory".

**Comment by edveal** [comment]
Here is a screen shot of the page I am having issues with. 

![Screenshot from 2022-01-19 17-29-38](https://user-images.githubusercontent.com/13770040/150235110-2111c16a-3737-4d84-84a0-44881cacb2e7.png)

**Comment by hgboldt** [comment]
Thanks, Ed!

Okay, so you have the directory created by the Gramps plugin. Copy that to your /var/www/html directory. Let's call that directory "gramps". That name goes into the "Directory" field. The "Instance Id" can be any identifier, using any letter or digit. That instance id is then specified in the shortcode.

A thought just occurred to me, though. The "Instance Id" is unnecessary. The directory name could be used to identify the Tangled Web instance, and could be specified in the shortcode instead of an "Instance Id". Does that make sense? 

Cheers! Hans

**Comment by edveal** [comment]
Yes, I think this makes sense. The directory that WPplugin is looking for is in _/var/www/html/**direcotyname**_.  I will try this out tonight after work.

**Comment by edveal** [comment]
FYI: That worked and I use a code based name for the "instance id". Your instructions call for the creation of a new page with the shortcode included. Before I dive into that, are you talking about a new html page on the server outside of WP or just creating a page via the WP Dashboard?

**Comment by hgboldt** [comment]
Create a page within WP. Within the body of the page, code `[tangled-web id='instance']` to display the gramps content.

**Comment by edveal** [comment]
That has worked and worked well. However, I ended up adding a "Featured Image" and after doing so I can't remove it and keep getting a json error when I try to update the Front Page. Suggestions? I am going to trash the page and recreate it without the "Featured Image". But thought you might like to know.

**Comment by hgboldt** [comment]
What is the specific error that you're getting on this? That sounds like a defect in the code. A screenshot might be helpful here.

**Comment by edveal** [comment]
Turns out it was a WP error and has nothing to do with TangledWeb. It is an error with the Block Editor on newer instances of WP.
